cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureStorageAccountTemplateBuilder.java

```diff
@@ -0,0 +1,67 @@
+package com.sequenceiq.cloudbreak.cloud.azure;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.azure.connector.resource.StorageAccountParameters;
+import com.sequenceiq.cloudbreak.cloud.exception.CloudConnectorException;
+import com.sequenceiq.cloudbreak.util.FreeMarkerTemplateUtils;
+
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+
+@Component
+public class AzureStorageAccountTemplateBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AzureStorageAccountTemplateBuilder.class);
+
+    @Value("${cb.arm.storageaccount.template.path:}")
```

Comment:
please double check if this parameter is required in redbeams' application.yml

Reply:
The storage account is created for holding the images of VMs and is used in case of freeipa and core (datalake / distrox) services. Redbeams does not create VMs so I have not added this parameter there.